### PR TITLE
AST-2158 - Fix earth icon opacity

### PR DESCRIPTION
**Pull request by neerajmasai**
### Description

- Improvement: earth icon visibility.
- css positioning improvements for globe icon.

### Screenshots

https://d.pr/i/6ON30p

### Types of changes

- Improvement: svg icon improved for better visibility.

### How has this been tested?

- Check svg icon on any color pallete it should be visible clearly against any color background or opacity.
### Checklist:
- [x] My code is tested
- [ ] My code passes the PHPCS tests
- [x] My code follows accessibility standards 
- [x] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [x] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: inc/lib/astra-notices/class-astra-notices.php [hunk 1]
```diff
@@ -116,7 +116,7 @@ public function dismiss_notice() {
 			}
 
 			if ( false === wp_verify_nonce( $nonce, 'astra-notices' ) ) {
-				wp_send_json_error( esc_html_e( 'WordPress Nonce not validated.', 'astra' ) );
+				wp_send_json_error( esc_html_e( 'WordPress Nonce not validated.' ) );
```
Comment (imnavanath) [inline inc/lib/astra-notices/class-astra-notices.php:119]:
```suggestion
				wp_send_json_error( esc_html_e( 'WordPress Nonce not validated.', 'astra' ) );
```

File: inc/customizer/custom-controls/assets/css/unminified/custom-controls-rtl.css [hunk 2]
```diff
@@ -2232,16 +2256,20 @@
...
+  left: 50%;
+  top: 50%;
+  color: white;
+  background: rgba(0,0,0,.6);
+  border-radius: 100%;
+  width: 16px;
+  height: 16px;
+  border: 1px solid rgba(0,0,0,.1);  
+} */
+
```
Comment (imnavanath) [inline inc/customizer/custom-controls/assets/css/unminified/custom-controls-rtl.css:2272]: Remove this commented CSS, after everything done run `grunt minify`

**Review by imnavanath — APPROVED** [review]

File: inc/lib/astra-notices/class-astra-notices.php (hunk 1 above)
Comment (neerajmasai) [reply inc/lib/astra-notices/class-astra-notices.php]: Done.

File: inc/customizer/custom-controls/assets/css/unminified/custom-controls-rtl.css (hunk 2 above)
Comment (neerajmasai) [reply inc/customizer/custom-controls/assets/css/unminified/custom-controls-rtl.css:2272]: oh, missed to remove this, will do that, thanks.

**Pull request merged**